packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/create_pin_page.dart

```diff
@@ -0,0 +1,142 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../app_extensions.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../bloc/create_pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+import '../services/create_pin_code_service.dart';
+
+class CreatePinPage extends StatefulWidget {
+  const CreatePinPage({
+    this.pinCodeArguments = const PinCodeArguments(
+      title: '',
+      isSessionTimeout: false,
+    ),
+    super.key,
+  });
+
+  final PinCodeArguments pinCodeArguments;
+
+  @override
+  State<CreatePinPage> createState() => _CreatePinPageState();
+}
+
+class _CreatePinPageState extends State<CreatePinPage> {
+  @override
+  void initState() {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      context.read<CreatePinBlocType>().events.temporaryDisableBiometrics(true);
```

Comment:
You are not passing a biometrics data source to `PinCodeKeyboard` why do you need to send this event?

Reply:
I have deleted this code.